src/univariate/continuous/noncentralchisq.jl

```diff
@@ -1,4 +1,4 @@
-"""
+doc"""
```

Comment:
`doc_str` doesn't play nice with documenter as a result #640 intentionally removes most of the `doc_str`s in the code and escapes everything accordingly.